connectors/aws/aws-dynamodb/src/main/java/io/camunda/connector/aws/dynamodb/operation/item/AddItemOperation.java

```diff
@@ -25,7 +25,7 @@ public AddItemOperation(final AddItem addItemModel) {
 
   public PutItemOutcome invoke(final DynamoDB dynamoDB) throws JsonProcessingException {
     String itemStr =
-        ObjectMapperSupplier.getMapperInstance().writeValueAsString(addItemModel.getItem());
+        ConnectorsObjectMapperSupplier.getCopy().writeValueAsString(addItemModel.getItem());
```

Comment:
Same here, a variable would be nicer, or to use the `DEFAULT_INSTANCE` static instance